composer.json

```diff
@@ -24,13 +24,13 @@
         "symfony/console": "^2.3 || ^3.0",
         "symfony/doctrine-bridge": "^2.2 || ^3.0",
         "symfony/framework-bundle": "^2.2 || ^3.0",
-        "symfony/security-acl": "^2.2 || ^3.0",
+        "symfony/security-acl": "^2.2 || ^3.0^",
         "sonata-project/exporter": "^1.3.1",
         "sonata-project/admin-bundle": "^3.0",
         "sonata-project/core-bundle": "^3.0"
     },
     "require-dev": {
-        "simplethings/entity-audit-bundle": "~0.1",
+        "simplethings/entity-audit-bundle": "^0.1",
```

Comment:
Could you try `0.1 - 0.9` and check if lowest and highest versions are correctly downloaded?

https://packagist.org/packages/simplethings/entity-audit-bundle#0.9.1